Application/EdFi.Admin.DataAccess/Utils/DefaultApplicationCreator.cs

```diff
@@ -11,20 +11,21 @@
 using EdFi.Admin.DataAccess.Models;
 using EdFi.Ods.Common;
 using EdFi.Ods.Common.Configuration;
+using Microsoft.Extensions.Configuration;
 
 namespace EdFi.Admin.DataAccess.Utils
 {
     public class DefaultApplicationCreator : IDefaultApplicationCreator
     {
-        private readonly IConfigValueProvider _configValueProvider;
+        private readonly IConfiguration _configuration;
         private readonly IUsersContextFactory _usersContextFactory;
 
         public DefaultApplicationCreator(
             IUsersContextFactory usersContextFactory,
-            IConfigValueProvider configValueProvider)
+            IConfiguration configuration)
```

Comment:
This breaks the net usage of this file. What you should have done is copied the file over, and wrapped it in NETSTANDARD and made this change. The original file should have been renamed to net.cs and wrapped in NETFRAMEWORK

Reply:
Created new file and wrapped inside NETFRAMEWORK